mapzen/whosonfirst/spatial/sqlite.py

```diff
@@ -0,0 +1,111 @@
+# THIS IS WET PAINT AND PROBABLY DOESN'T WORK YET
+# (20180416/thisisaaronland)
+
+import mapzen.whosonfirst.spatial
+import time
+import logging
+import sqlite3
+import json
+
+class spatialite(mapzen.whosonfirst.spatial.base):
+
+    def __init__(self, **kwargs):
+
+        dsn = kwargs.get("dsn", "/Users/stephen.epps/Desktop/sqlite-db/whosonfirst-data-latest.db")
```

Comment:
@thisisaaronland what's the recommendation on generalizing this instead of hard coding the file path to the sqlite db file?